model/userdata_test.go

```diff
@@ -0,0 +1,108 @@
+package model
+
+import (
+	"github.com/stretchr/testify/assert"
+
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+const (
+	s3Body = `#cloud-config
+---
+coreos: {}
+`
+	USERDATA_S3       = "s3"
+	USERDATA_INSTANCE = "instance"
```

Comment:
Should we define these in `userdata.go` instead so that we can use these to avoid hard-coded strings "s3" and "instance"?